multiversx_sdk_cli/validators/validators_file.py

```diff
@@ -34,6 +35,19 @@ def load_signers(self) -> List[ValidatorSigner]:
 
         return signers
 
+    def load_public_keys(self) -> List[ValidatorPublicKey]:
+        public_keys: List[ValidatorPublicKey] = []
+
+        for validator in self.get_validators_list():
+            # Get path of "pemFile", make it absolute
+            validator_pem = Path(validator.get("pemFile")).expanduser()
```

Comment:
it should be clear for the user, the format of the validators file, that if should have `pemFile`. Maybe to document it somehow in README or in command help, or to make reference to the example file that we have

Reply:
I agree. As of now, it is documented in the [docs](https://docs.multiversx.com/validators/staking/#the-staking-transaction), but in the future maybe we'll add a dedicated section in the cookbook.